packages/dev/core/src/FlowGraph/flowGraphContext.ts

```diff
@@ -96,6 +61,11 @@ export class FlowGraphContext {
      */
     private _executionId = 0;
 
+    /**
+     * A mapping of paths to target objects
+     */
+    pathMap: Map<string, any> = new Map();

```

Comment:
nit - a simple key-value object will suffice. Unless you are planning on using the maps' functionality?